server/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const path = require('path');
+const cluster = require('cluster');
+const numCPUs = require('os').cpus().length;
+const isDev = process.env.NODE_ENV !== 'production';
+const PORT = process.env.PORT || 5000;
+var bodyParser = require('body-parser');
+
+// Multi-process to utilize all CPU cores.
+if (!isDev && cluster.isMaster) {
+  console.error(`Node cluster master ${process.pid} is running`);
+
+  // Fork workers.
+  for (let i = 0; i < numCPUs; i++) {
+    cluster.fork();
+  }
+
+  cluster.on('exit', (worker, code, signal) => {
+    console.error(
+      `Node cluster worker ${worker.process.pid} exited: code ${code}, signal ${signal}`
+    );
+  });
+} else {
+  const app = express();
+
+  // Priority serve any static files.
+  app.use(express.static(path.resolve(__dirname, '../client/build')));
+  app.use(express.json());
+
+  app.listen(PORT, function () {
+    console.error(
+      `Node ${
+        isDev ? 'dev server' : 'cluster worker ' + process.pid
+      }: listening on port ${PORT}`
+    );
+  });
```

Comment:
This code part can be relocated very well in my opinion. Then the ``index.js`` stays clean.

Reply:
IMO it should stay in the index file

Reply:
Okay then I will leave it in the index file

Reply:
My thought was to simply outsource it into a function.